### PR TITLE
Fix conditional dependencies for wheels in setup.py

### PR DESCRIPTION
**Pull request by hackebrot**
Related to #557

cc @ionelmc

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -71,6 +62,14 @@
...
     include_package_data=True,
     install_requires=requirements,
+    extras_require={
+        ':sys_platform=="win32" and python_version=="2.7"': [
+            'PyYAML>=3.10'
+        ],
+        ':sys_platform!="win32" or python_version!="2.7"': [
+            'ruamel.yaml>=0.10.12'
+        ]
+    },
```
Comment (ionelmc) [inline setup.py]: This doesn't install anything on *nix and py2.7. Intended?

Comment (hackebrot) [inline setup.py]: Uhmhmm

```
':sys_platform!="win32" or python_version!="2.7"'
```
anything on *nix and py2.7 > True or False > True

Comment (ionelmc) [inline setup.py]: Wat. You're right. :grin:

Comment (hackebrot) [inline setup.py]: :stuck_out_tongue_closed_eyes:

**Comment by hackebrot** [comment]
:bell:

**Comment by hackebrot** [comment]
Rebased :triumph:

**Comment by hackebrot** [comment]
Think we should get this in as long as we don't have a better solution to the whole YAML issue.

**Pull request merged**